File: pangram/pangram.java
class Solution {
  public boolean checkIfPangram(String sentence) {
    // TODO: Write your code here
   Set<Character> seen = new HashSet<>();

   for (int i = 0; i <sentence.length(); i++) {
    char currChar = Character.toLowerCase(sentence.charAt(i));

    if(Character.isLetter(currChar)) {
      seen.add(currChar);
    }
   }
    return seen.size() == 26;
  }
}

File: pangram/pangram.py
class Solution:
  def checkIfPangram(self, sentence):
    # TODO: Write your code here
    seen = set()

    for i in range(len(sentence)):

      currChar = sentence[i].lower()
      if currChar.isalpha():
        seen.add(currChar)
    return len(seen) == 26
    # return False
